lib/fastlane/plugin/revenuecat_internal/actions/bump_version_update_changelog_create_pr_action.rb

```diff
@@ -116,7 +116,8 @@ def self.available_options
                                        env_name: "RC_INTERNAL_FILES_TO_UPDATE_ON_LATEST_STABLE_RELEASES",
                                        description: 'Hash of files that contain the version number and only need to' \
                                                     'be updated on stable releases (no prereleases) and on the latest' \
-                                                    'major (no hotfixes).' \
+                                                    'major (no hotfixes). Note that the version will be updated as' \
+                                                    'long as it matches a semver stable version, even if its not the previous version' \
```

Comment:
should all of these have spaces at the end of every sentence? I don't think ruby would auto-add spaces. Feels like we've had this for all actions

Reply:
Good catch! Yeah that's right 👍